build.gradle

```diff
@@ -15,8 +15,7 @@ plugins {
 }
 
 group 'bio.terra'
-version '0.0.5-SNAPSHOT'
-
+version '0.0.6-SNAPSHOT'
```

Comment:
No, there is nothing to prevent that. Artifactory holds both versions and serves the most recent. We really need some version management scheme for this and the datarepo-client.
I looked at a promising plug-in, but I couldn't figure out how it would fit with our workflow.

I'll make a ticket to track the problem and keep doing background work to find a workable scheme. Its not like we are the only ones who do this!